lib/modules/apostrophe-login/index.js

```diff
@@ -160,25 +160,26 @@ module.exports = {
     // If the user's login SUCCEEDS, the return value is
     // the `user` object.
 
-    self.verifyLogin = async function(username, password) {
+    self.verifyLogin = async function(username, password, done) {
```

Comment:
I understand this change was needed to make it work, but please instead change the code that invokes this method to handle the icky callback-ness, allowing this method to follow the standard async pattern and just return a user or throw an exception nicely. So far as written an exception would probably not behave as expected.

Reply:
Right now this method is probably passed directly to passport, but instead a wrapper function that invokes it and handles the callback-itude should be passed.